qipfs/filestore.go

```diff
@@ -87,7 +87,7 @@ func NewFilesystem(ctx context.Context, cfgMap map[string]interface{}) (qfs.File
 
 	node, err := core.NewNode(ctx, &cfg.BuildCfg)
 	if err != nil {
-		return nil, fmt.Errorf("error creating ipfs node: %s", err.Error())
+		return nil, fmt.Errorf("qipfs: error creating ipfs node: %s", err.Error())
```

Comment:
this should be an error wrap

```suggestion
		return nil, fmt.Errorf("qipfs: error creating ipfs node: %w", err.Error())
```